docs/tmp-env-setup.md

```diff
@@ -0,0 +1,29 @@
+With the new microservices shipping, the old [environment-setup](environment-setup) document might not work. In the transition, this document may be helpful for developers. Here are some instructions for different scenarios, they are based on [next.config.js](../src/web/next.config.js) code logic. This `next.config.js` file configures which `env` file to use. It reads `.env` in the root first, if there is no `.env` in the root, it reads [env.development](../config/env.development).
+
+- Want to run back-end services locally, and to have front-end using the services.
+
+  This is the default setting, you **don't need to copy or modify** any env file. Just run `npm run services:start` and `npm run dev`. However, if it doesn't work and you have old `.env` that was copied from old `.env.example`, delete it.
+
```

Comment:
We should add a blurb here, all commands should be run in the root of the telescope project. 

"If any of the commands below commands are failing, use the command `pwd` to find your current directory and navigate back to project root or provide an example of it here"

Reply:
Yes, I personally think we should go with that.